### DISCUSSION TITLE
Need help with Nextjs forcing Axios to use https

### DISCUSSION THREAD

**Discussion by trevorbidlack** [vercel/next.js#19883] [Help] [answered]
I have a specific use case that requires me to use http instead of https for an api I am fetching with Axios. Every time I send a request using http as the url prefix in Axios, it sends it as https and I believe it has come down to this meta tag:
`<meta http-equiv="Content-Security-Policy" content="upgrade-insecure-requests">`

![image](https://user-images.githubusercontent.com/37959576/101268888-9e634680-3736-11eb-8355-d30ff878d79b.png)

Can I override this default meta tag from Nextjs? I figure I could use next/head or something to do so, but have had no luck playing around with it.

**Comment by jamesmosier** [comment, marked as answer]
That is not a meta tag that is injected by Next. That must be coming from another script or plugin (or maybe even your server? 🤔)  you are importing.

You can set a CSP like this: https://github.com/vercel/next.js/issues/256#issuecomment-679591805
